core/templates/pages/facilitator-dashboard-page/facilitator-dashboard-page.mainpage.html

```diff
@@ -1,46 +0,0 @@
-<!DOCTYPE html>
-<html ng-app="oppia" itemscope itemtype="http://schema.org/Organization">
-  <head>
-    @load('base-components/header.template.html', {"title": "Facilitator Dashboard | Oppia"})
-    <script>
-      // This prevents angular pages to cause side effects to hybrid pages.
-      // TODO(#13080): Remove fake resumeBootstrap once angularjs is removed
-      // from this page.
-      if (!window.angular.resumeBootstrap) {
-        window.angular.resumeBootstrap = function(extraModules) {};
-      }
-    </script>
-  </head>
-  <body>
-    <oppia-root>
-      <div ng-controller="Base" dir="<[direction]>">
-        <oppia-base-content>
-          <navbar-breadcrumb>
-            <ul class="nav navbar-nav oppia-navbar-breadcrumb">
-              <li>
-                <span class="oppia-navbar-breadcrumb-separator"></span>
-                <span translate="I18N_TOPNAV_FACILITATOR_DASHBOARD"></span>
-              </li>
-            </ul>
-          </navbar-breadcrumb>
-
-          <content>
-            <oppia-facilitator-dashboard-page></oppia-facilitator-dashboard-page>
-          </content>
-
-          <page-footer></page-footer>
-        </oppia-base-content>
-      </div>
-
-    </oppia-root>
-    @load('pages/footer_js_libs.html')
-    <script>
-      // Use the normal default browser eventlistener without passive
-      // set to false.
-      // TODO(#10749): Remove workaround for passive event listeners when
-      // jQuery supports it.
-      EventTarget.prototype.addEventListener = (
-        EventTarget.prototype.addEventListener._original);
```

Comment:
Do we need to move this to the new root page?

Reply:
Yes, if the original page used passive event listeners, the migrated one should also. Thanks.

Done